test/resources/playbooks/docker/create.yml

```diff
@@ -132,7 +132,7 @@
         exposed_ports: "{{ item.exposed_ports | default(omit) }}"
         published_ports: "{{ item.published_ports | default(omit) }}"
         ulimits: "{{ item.ulimits | default(omit) }}"
-        networks: "{{ item.networks | default(omit) }}"
+        networks: "{{ (item | molecule_get_docker_container_networks) or omit }}"
```

Comment:
I have the impression that this does not work well "or omit" does not sounds like a valid jinja way to  do it. I think that you still need a `| default(omit)`